src/topojson/topojson_intf.ml

```diff
@@ -241,3 +241,29 @@ module type Topojson = sig
       parser and constructor. *)
   module Make (J : Json) : S with type json = J.t
 end
+
+module type Accessor = sig
```

Comment:
Same point as above, let's remove the wrapping `Accessor` module so it is more direct to use these functions

Reply:
If i remove 

```ocaml
module type Accessor = sig
  type t
```

I get this error ```Value declarations are only allowed in signatures``` for these parts of the code
```ocaml
val get_point :
    Geometry.geometry -> (Geometry.Point.t, [> `Msg of string ]) result
  val get_point_exn : Geometry.geometry -> Geometry.Point.t
```

Reply:
Ah I didn't see they weren't with the Geometry module type, if you move them up there that would be better. You won't need to redeclare `Geoemetry` either so your functions become:

```ocaml
val get_point : geometry -> (Geometry.Point.t, [> `Msg of string ]) result

  val get_point_exn : geometry -> Point.t
```